limits.go

```diff
@@ -0,0 +1,75 @@
+package libp2p
+
+import (
+	"github.com/libp2p/go-libp2p/p2p/host/autonat"
+	circuitv1 "github.com/libp2p/go-libp2p/p2p/protocol/circuitv1/relay"
+	circuitv2 "github.com/libp2p/go-libp2p/p2p/protocol/circuitv2/relay"
+	"github.com/libp2p/go-libp2p/p2p/protocol/holepunch"
+	"github.com/libp2p/go-libp2p/p2p/protocol/identify"
+	"github.com/libp2p/go-libp2p/p2p/protocol/ping"
+
+	rcmgr "github.com/libp2p/go-libp2p-resource-manager"
+)
+
+// SetDefaultServiceLimits sets the default limits for bundled libp2p services
+func SetDefaultServiceLimits(limiter *rcmgr.BasicLimiter) {
```

Comment:
Now that we check if the value already exists, this doesn't need to be exported any more.

Reply:
I think we should keep it exported so that the user can call it on his own limiter.